### PR TITLE
Issue #13213: Remove unnecessary comments

### PR DESCRIPTION
**Pull request by Pinti0001**
Issue #13213 
Removed // ok from InputJavadocPackageHtmlIgnored2.java and  InputJavadocTypeNoJavadocOnInterface.java and InputJavadocVariableInner2.java with its suppressions

### CODE REVIEW

File: config/checkstyle-input-suppressions.xml [hunk 1]
```diff
@@ -1047,12 +1043,6 @@
             files="checks[\\/]javadoc[\\/]javadoctype[\\/]InputJavadocTypeTags.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]javadoc[\\/]javadoctype[\\/]InputJavadocTypeTags.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]javadoc[\\/]javadocvariable[\\/]InputJavadocVariableInner2.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]javadoc[\\/]javadocvariable[\\/]InputJavadocVariableInner2.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]javadoc[\\/]javadocvariable[\\/]InputJavadocVariableInner2.java"/>
```
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:1055]: Everything lgtm....

**Review by MANISH-K-07 — COMMENTED** [review]
The CI are all green, changes all good. Will be approved by maintainers ;)

File: config/checkstyle-input-suppressions.xml (hunk 1 above)
Comment (Pinti0001) [reply config/checkstyle-input-suppressions.xml:1055]: Thanks

**Review by nrmancuso — APPROVED** [review]
@Pinti0001 thanks a lot!

**Pull request merged**